style.css

```diff
@@ -283,9 +291,11 @@ input[type=submit]:hover{
 	box-sizing: border-box;
 	background-color: green;
 	margin-top: 3.5vw;
+	margin-left: 12vw;
 	padding-left: 2vw;
 	height: 40vw;
-	grid-column: 6;
```

Comment:
I have moved the login box to the center as it makes more sense there rather than on the right where users will not initially look